Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,411 @@
+        readable_output=tableToMarkdown(
+            f"DSL Search: {str(args.get('query'))}", records
+        ),
+    )
+
+    return results
+
+
+def search_domain_with_dsl_statistics(client: Client, **args) -> CommandResults:
+    params = {
+        "include_ips": args.get("include_ips", "false"),
+        "page": int(args.get("page", 0)),
+        "scroll": str(args.get("scroll", "false")),
+    }
+
+    body = {"query": str(args.get("query"))}
+
+    uri = f"/domains/stats"
+    response = client._http_request("POST", uri, params=params, json_data=body)
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.domain_dsl_search",
+        outputs={"domain_statistics": response},
+        readable_output=tableToMarkdown(
+            f"Statistics for Query: {str(args.get('query'))}", response
+        ),
+    )
+
+    return results
+
+
+def search_ip_with_dsl(client: Client, **args) -> CommandResults:
+    params = {"page": int(args.get("page"))}
+
+    # Docs: https://docs.securitytrails.com/docs/how-to-use-the-dsl
+    # Example: ptr_part = 'stackoverflow.com'
+    body = {"query": str(args.get("query"))}
+
+    uri = f"/ips/list"
+    response = client._http_request("POST", uri, params=params, json_data=body)
+    records = response.get("records")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.ip_dsl_search",
+        outputs={"dsl_search": records},
+        readable_output=tableToMarkdown(
+            f'DSL Search: {str(args.get("query"))}', records
+        ),
+    )
+
+    return results
+
+
+def search_ip_with_dsl_statistics(client: Client, **args) -> CommandResults:
+    # Docs: https://docs.securitytrails.com/docs/how-to-use-the-dsl
+    # Example: ptr_part = 'stackoverflow.com'
+    body = {"query": str(args.get("query"))}
+
+    uri = f"/ips/stats"
+    response = client._http_request("POST", uri, json_data=body)
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.ip_dsl_statistics",
+        outputs={"dsl_statistics": response},
+        readable_output=tableToMarkdown(
+            f"DSL Statistics: {str(args.get('query'))}", response
+        ),
+    )
+
+    return results
+
+
+def get_ip_whois(client: Client, **args) -> CommandResults:
+    ip = args.get("ip")
+
+    uri = f"/ips/{ip}/whois"
+    response = client._http_request("GET", uri)
+    records = response.get("record")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.ip_whois",
+        outputs={"ip_whois": records},
+        readable_output=tableToMarkdown(f"Whois for {ip}", records),
+    )
+
+    return results
+
+
+def get_ip_useragents(client: Client, **args) -> CommandResults:
+    ip = args.get("ip")
+
+    params = {"page": int(args.get("page"))}
```

Comment:
mypy is failing this line, Please add a default value like the rest of the commands

Reply:
added default value of 1 for page